community/modules/compute/pbspro-execution/main.tf

```diff
@@ -68,7 +68,8 @@ module "execution_startup_script" {
 }
 
 module "pbs_execution" {
-  source = "github.com/GoogleCloudPlatform/hpc-toolkit//modules/compute/vm-instance?ref=v1.35.0&depth=1"
+  # source = "github.com/GoogleCloudPlatform/hpc-toolkit//modules/compute/vm-instance?ref=v1.35.0&depth=1"
+  source = "../../../../modules/compute/vm-instance"
```

Comment:
This is not going to work reliably unfortunately.

Reply:
I asked in chat when I sent out this PR, but I'll ask here too: do we have guidelines or a standard strategy for referencing changes made in the same PR?

Update the reference in a separate PR and change the ref to "develop"?  Use relative references?  Or do we need to wait for the next release to update the reference in the "calling" module?

Reply:
Since this issue is only occurring in the PBS Pro and Chrome Remote Desktop modules, I reverted the change there.  We can make this change after the next release, so the ref can be updated to the next label.